treeherder/webapp/api/performance_data.py

```diff
@@ -471,6 +471,11 @@ class PerformanceAlertSummaryViewSet(viewsets.ModelViewSet):
     ordering = ('-created', '-id')
     pagination_class = AlertSummaryPagination
 
+    def update(self, request, *args, **kwargs):
+        instance = self.get_object()
+        instance.update_status()
+        return super().update(request, *args, **kwargs)
```

Comment:
```suggestion
        serializer = self.get_serializer(instance)
        return Response(serializer.data)
```
Calling super().update is not necessary, since the summary instance does not need to update itself with the request data sent from the frontend.